internal/uvm/create.go

```diff
@@ -389,3 +391,42 @@ func (uvm *UtilityVM) CloseGCSConnection() (err error) {
 	}
 	return
 }
+
+// parseStorageSpaceEVDPath verifies the host path provided for the mount
+// type "ExtensibleVirtualDisk" and ExtensibleVirtualDiskType "space"
+// The correct path is in the "evd://space/{pool_guid}{disk_guid}" format.
+func parseStorageSpaceEVDPath(path string) error {
+	if !strings.HasPrefix(path, `evd://space/{`) ||
+		!strings.HasSuffix(path, "}") {
+		return errors.New("storage space evd path must follow 'evd://space/{pool_guid}{disk_guid}' format")
+	}
+	pool_guid_start := len(`evd://space/{`)
+	pool_guid_end := pool_guid_start + 36 // length of GUID string
+	disk_guid_start := pool_guid_end + 2  // skips '}{'
+	disk_guid_end := disk_guid_start + 36
+	pool_guid := path[pool_guid_start:pool_guid_end]
+	disk_guid := path[disk_guid_start:disk_guid_end]
+	if _, err := guid.FromString(pool_guid); err != nil {
```

Comment:
I hate to suggest this, but do we think using regex would be clearer here?

Reply:
The issue is that we are not trying to match with one of the many possible formats. It has to be the exact same format (with just all possible characters for the guid part itself). So we can use a regex here but it will be much costlier than this simple verification. However, I agree that simple code should be preferred over complex (/optimized) code. So I can switch this to a regex if everyone feels that way.

Reply:
Resolving as this entire function is removed.